android-extra.config

```diff
@@ -0,0 +1,179 @@
+#  KEEP ALPHABETICALLY SORTED
```

Comment:
Iirc I asked this before, `android-extra.config` is not an "android" config (https://android.googlesource.com/kernel/configs/+/refs/tags/android-11.0.0_r28/r/android-4.19/) but something we as SODP add on top, shared by all devices.

Suggestion is to rename it to `sony/sodp-base.config`.

Reply:
+1 to this.

Reply:
@pablomh I don't mind either, as long as it is consistent :ok_hand: 

The kernel uses `<snake_case_target>_defconfig`, AOSP uses `<file>.config` for the extra bits.